lib/assert.js

```diff
@@ -95,43 +93,22 @@ function innerFail(obj) {
 }
 
 /**
- * @param {any} actual
- * @param {any} expected
- * @param {string | Error} [message]
- * @param {string} [operator]
- * @param {Function} [stackStartFn]
+ * Throws an AssertionError with the given message.
+ * @param {any | Error} [message]
```

Comment:
```suggestion
 * @param {Error} [message]
```

Reply:
The `message` is not limited to just being an `Error`. It can be any stringifiable value or an Error